app/components/participant-creator/component.js

```diff
@@ -73,6 +73,12 @@ export default Ember.Component.extend({
             });
             if (!useAsPassword) {
                 password = makeId(10);
+                this.get('extra').pushObject({
+                    key: 'password',
+                    value: password
+                });
+                extra['password'] = password;
+                this.set('useAsPassword', 'password');
```

Comment:
This parameter is used like a boolean in the template, but the value is just the string "password".

(it's neither a boolean, nor an actual password)

Is this intentional?